tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    pulse = Pulse.ConstantPulse(duration, 2*np.pi, 0.*2*np.pi, 0)
</pre><p>The <code>0*2*np.pi</code> seems unnecessary, don't you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>